src/classes/ADDR_CannotDelete_TEST.cls

```diff
@@ -0,0 +1,451 @@
+/*
+    Copyright (c) 2020, Salesforce.org
+    All rights reserved.
+    
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+ 
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group Addresses
+* @group-content ../../ApexDocContent/Addresses.htm
+* @description Tests for ADDR_CannotDelete_TDTM.
+*/
+
+@isTest
+private class ADDR_CannotDelete_TEST {
+    /*********************************************************************************************************
+    * @description Retrieves the record type Ids for Household and Administrator Accounts. 
+    */
+    public static String hhAccRecordTypeID = UTIL_Describe.getHhAccRecTypeID(); 
+    public static String adminAccRecordTypeID = UTIL_Describe.getAdminAccRecTypeID();
+
+    /*********************************************************************************************************
+    * @description Test method to test that when Prevent_Address_Deletion__c is enabled in Hierarchy Settings, and
+    * Address has a child Account record associated to it, the Address record cannot be deleted.
+    */
+    @isTest 
+    private static void addrCannotDeleteWithAccount() {
```

Comment:
@m-kinkade My apologies for missing this the first time. Consider removing the 'addr' prefix for these methods as we have context from the class these are running in.